server/dash/dashboards/neighborhood_recent.py

```diff
@@ -209,12 +255,30 @@ def update_figure(selected_council):
 
 
 @app.callback(
-    Output("pie_graph", "pie_fig"),
+    Output("pie_graph", "figure"),
     Input("council_list", "value")
 )
 def update_council_figure(selected_council):
+    """Dash Callback Function to update the Request Type Pie Chart at the middle of the dashboard.
+
+    This function takes the selected neighborhood council (nc) value from the "council_list" dropdown and 
+    outputs the the pie chart showing the share of each request types. The pie chart will
+    show up inside dcc.Graph object as long as id "pie_graph" is passed in.
+
+    Typical usage example:
+
+        html.Div(
+                    dcc.Graph(id='pie_graph', 
+                    ...
+                )
+    """
     pie_df = df.query(f"councilName == '{selected_council}' and createdDate >= '{start_date}'").groupby(['typeName']).agg('count').reset_index()  # noqa
 
+    # The following check is to ensure Dash graphs are populated with dummy data when query returns empty dataframe.
+    if pie_df.shape[0] == 0:
+        pie_df = pd.DataFrame(columns=["srnumber", "typeName"])
+        for i, request_type in enumerate(DISCRETE_COLORS_MAP):
+            pie_df.loc[i] = [1, request_type]
```

Comment:
Gotcha, I saw your other comment about using 0. Can you open a new issue and assign it to the designers? We should get their input to see whether we should have no pie chart or a pie chart with equal parts when there are no requests.

It doesn't need to block this PR, though.

Reply:
Sounds good. Will open on e and assign to designers.